packages/enzyme/src/ReactWrapper.js

```diff
@@ -374,9 +375,13 @@ class ReactWrapper {
    * @returns {Boolean}
    */
   contains(nodeOrNodes) {
+    const isArray = Array.isArray(nodeOrNodes);
+    const rstNodeOrNodes = isArray
+      ? nodeOrNodes.map(getAdapter().elementToNode)
+      : getAdapter().elementToNode(nodeOrNodes);
     const predicate = Array.isArray(nodeOrNodes)
-      ? other => containsChildrenSubArray(nodeEqual, other, nodeOrNodes)
-      : other => nodeEqual(nodeOrNodes, other);
+      ? other => containsChildrenSubArray(nodeEqual, other, rstNodeOrNodes)
```

Comment:
probably at this point, rather than two paired multiline ternaries, this would be better as an `if`/`else` branch.